tests/utils/chain.go

```diff
@@ -556,28 +553,29 @@ func NewSignatureAggregator(apiUri string, subnets []ids.ID) *aggregator.Signatu
 	trackedSubnets := set.NewSet[ids.ID](len(subnets))
 	trackedSubnets.Add(subnets...)
 	registry := prometheus.NewRegistry()
+	messageCreator, err := message.NewCreator(
+		logging.NoLog{},
+		registry,
+		constants.DefaultNetworkCompressionType,
+		constants.DefaultNetworkMaximumInboundTimeout,
+	)
+	Expect(err).Should(BeNil())
+
 	appRequestNetwork, err := peers.NewNetwork(
-		logging.Info,
+		logging.Error,
```

Comment:
(no action necessary) I assume that this is because the network now spews a lot more with P2P changes. Is `Warning` level also too loud?

Reply:
I changed this because the signature aggregator is not under test in this repo's e2e test suites, so the logs seemed unnecessary.